### PR TITLE
EN-2264-PAM-widgets-data-table-task-list-show-diagram-fix

### PR DESCRIPTION
**Pull request by alepintus**
Hi @joewhite101 ,
this is the cherry-pick from [this PR](https://github.com/entando/entando-components/pull/415).
Could you please take a look?

Thanks!

### CODE REVIEW

**Comment by entando-service** [comment]
All is Well

File: plugins/entando-plugin-jpkiebpm/src/main/java/org/entando/entando/plugins/jpkiebpm/aps/system/services/kie/KieFormManager.java [hunk 1]
```diff
@@ -1226,8 +1230,12 @@ public String completeTask(KieBpmConfig config, String payload, String container
 
     @Override
     public JSONObject getTaskDetails(KieBpmConfig config, String taskId) throws ApsSystemException {
+        
+        
+        logger.info("---------------------------------------------");
 
-        logger.info("getTaskDetails(taskId: {}", taskId);
+        logger.info("getTaskDetails (config: {}", config.toString());
```
Comment (joewhite101) [inline plugins/entando-plugin-jpkiebpm/src/main/java/org/entando/entando/plugins/jpkiebpm/aps/system/services/kie/KieFormManager.java]: Debug

File: plugins/entando-plugin-jpkiebpm/src/main/java/org/entando/entando/plugins/jpkiebpm/aps/system/services/kie/KieFormManager.java [hunk 2]
```diff
@@ -400,22 +400,26 @@ public KieTaskDetail getTaskDetail(KieBpmConfig config, String containerId, fina
...
+    public String getProcInstDiagramImage(KieBpmConfig config, String containerId) throws ApsSystemException {
         return getProcInstDiagramImage(config,
-                 containerId, processId,
+                 containerId, 
                  null);
     }
 
     @Override
-    public String getProcInstDiagramImage(KieBpmConfig config, String containerId, String processId, Long pInstanceId) throws ApsSystemException {
+    public String getProcInstDiagramImage(KieBpmConfig config, String containerId,  String pInstanceId) throws ApsSystemException {
```
Comment (joewhite101) [inline plugins/entando-plugin-jpkiebpm/src/main/java/org/entando/entando/plugins/jpkiebpm/aps/system/services/kie/KieFormManager.java]: Do we have sufficient test coverage on this method? I see no major structural changes but if we don't have a test lets add one now

Comment (alepintus) [reply plugins/entando-plugin-jpkiebpm/src/main/java/org/entando/entando/plugins/jpkiebpm/aps/system/services/kie/KieFormManager.java]: Nope, I'll create a new test for this method

**Pull request merged**